admin/app/controllers/admin/ReaderRevenueAdminController.scala

```diff
@@ -48,7 +48,7 @@ class ReaderRevenueAdminController(wsClient: WSClient, val controllerComponents:
         ) { region: ReaderRevenueRegion =>
           val requester: String =
             UserIdentity.fromRequest(request) map (_.fullName) getOrElse "unknown user (dev-build?)"
-          val millisSinceEpoch = DateTime.now.getMillis()
+          val millisSinceEpoch = Instant.now().toEpochMilli()
```

Comment:
@tomrf1 @GHaberis unrelated to this PR but do you think we could move this redeploy functionality out of frontend now? Maybe it makes more sense to live in your tooling and frontend just checks the S3 bucket for the date?

Reply:
good question, and yes we've already built a replacement on our side.
I'll add a trello card to delete this